README.md

```diff
@@ -190,6 +190,28 @@ CSS selector:
 }
 ```
 
+The appearance of options can be configured by using the parametr `dropdownClassName` for element `<szn-select>` (e.g. <szn-select dropdownClassName="dropdown-class-name">...) and the following
```

Comment:
1. Prosím max. 80 znakov na riadku
2. `dropdownClassName` je DOM vlastnosť, nie atribút, takže ukážka kódu je neplatná.
3. Formulácia a gramatika :). Skús to takto (toto je návrh, oponuj, prosím, čo mám zle alebo nezrozumiteľne):

The appearance of the options dropdown can be customized by setting the `dropdownClassName` property of the `<szn-select>` element:

```html
<szn-select id="my-select">
  ...
</szn-select>
<script>
  document.getElementById('my-select').dropdownClassName = 'my-custom-dropdown'
</script>
```

This will configure this particular `<szn-select>` to set the `my-custom-dropdown` CSS class on its own options dropdown (the `<szn-select--options>` element).

With the custom CSS class set, your will now be able to override the look&feel of this particular select's dropdown without affecting the rest. One approach you may utilize is overriding the CSS custom properties configuring the look&feel - if you are not concerned about compatibility with Internet Explorer. The code below shows the default configuration specified in `szn-select.css`:

```css
.my-custom-dropdown {
  --szn-select--options--border-width: var(--szn-select--border-width);
  --szn-select--options--border-color: var(--szn-select--border-color);
  --szn-select--options--border-radius-dropdown: 0 0 var(--szn-select--border-radius) var(--szn-select--border-radius);
  --szn-select--options--border-radius-dropup: var(--szn-select--border-radius) var(--szn-select--border-radius) 0 0;
  --szn-select--options--border-radius-mutli-select: var(--szn-select--border-radius);
  --szn-select--options--item-padding: var(--szn-select--option-padding);
  --szn-select--options--item-indent: var(--szn-select--option-indent);
  --szn-select--options--text-color: var(--szn-select--text-color);
  --szn-select--options--disabled-text: var(--szn-select--disabled-text-color);
  --szn-select--options--background-color: var(--szn-select--background);
  --szn-select--options--selected-background: var(--szn-select--selected-option-background);
  --szn-select--ui--dropdown-offset: calc(-1 * var(--szn-select--border-width));
  --szn-select--ui--dropdown-min-width: var(--szn-select--min-width);
}
```

Reply:
Vim, ze ma anglictina je na nizke az nulove urovni :-(, takze s Tvym navrhem souhlasim. Diky

Reply:
Nebuď na seba tak tvrdý :). A ja tiež robím chyby. Len to, prosím, aspoň prejdi či Ti to prejde zrozumiteľné. Ak áno, uprav, prosím, tento merge request.

Reply:
upraveno